src/v2/components/MessageList/MessageList.tsx

```diff
@@ -336,7 +336,7 @@ const MessageListWithContext = <
    * we set the loading state one cycle behind to ensure the messages are set before the
    * change to the loading state is registered.
    */
-  const [messagesLoading, setMessagesLoading] = useState(false);
+  const [messagesLoading, setMessagesLoading] = useState(true);
```

Comment:
seems like the comment above this said this was false for a reason

Reply:
Do you remember what was the reason? Because its causing LoadingIndicator to not render on ChannelScreen